src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -86,4 +152,40 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     toggleSpinner() {
         this.showSpinner = !this.showSpinner;
     }
+
+    getDisplayedFieldsMappedByAPIName(sectionsList) {
+        let allFields = {};
+        sectionsList.forEach(section => {
+            const fields = section.getAllFieldsByAPIName();
+
+            allFields = Object.assign(allFields, fields);
```

Comment:
no need to change but FYI this is the same as `allFields = {...allFields, ...fields };`